frontend/src/assets/fonts/README.md

```diff
@@ -1,19 +1,25 @@
 # Export PDF Language Support
 
-To support multiple languages with special characters we need to register fonts and set the fontFamily (example in ExportContent.tsx)
-Download font families tts or otf files from https://fonts.google.com/noto to assets/fonts/
+To support multiple languages with special characters we need to register fonts and set the font-family -
+see [ExportContent.tsx](../../components/ExportChat/ExportContent.tsx). You can download additional font families
+(ttf or otf) from https://fonts.google.com/noto and place in src/assets/fonts.
 
-Currently can only use a single file at a time, so we can merge multiple using script from https://github.com/notofonts/nototools/blob/main/nototools/merge_fonts.py.
+Currently only a single font-family can be used at a time, so multiple fonts can be merged using this script:
+https://github.com/notofonts/nototools/blob/main/nototools/merge_fonts.py.
 
 The current CombinedFont.ttf contains:
 
 - NotoSans-Regular.ttf
-- NotoSerifDevanagari-Regular.ttf
 - NotoKufiArabic-Regular.ttf
+- NotoSansHebrew-Regular.ttf
 - NotoSansThai-Regular.ttf
 - NotoSerifBengali_Condensed-Regular.ttf
+- NotoSerifDevanagari-Regular.ttf
 - NotoSerifGurmukhi-Regular.ttf
-- NotoSansHebrew-Regular.ttf
+
+Japanese and Chinese font packs are not included in the CombinedFonts family as they are prohibitively large, so if you are
```

Comment:
Seems weird to mention Japanese and Chinese specifically. We're missing korean too, non? I suppose Japanese and Chinese might be more likely to come up...

Reply:
I know, it's all a bit random to me, but I think it was a work-in-progress that just got kicked into the long grass.

I would be tempted to do as you suggest, and remove the chinese / japanese font files from the repo, plus the commented out lines, and augment the README to say how to use different export fonts.

Reply:
Done.